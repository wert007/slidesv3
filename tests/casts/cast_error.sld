func main() {
    let e : int? = none;
    // This could have a better error message
    let d : int = cast e : int;
}