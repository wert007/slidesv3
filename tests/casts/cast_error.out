Error in ..\tests\casts\cast_error.sld:4:19: No cast necessary between types int? and int. Remove the unnecessary cast. Note, that cast will return int?.
Error in ..\tests\casts\cast_error.sld:4:24: Cannot convert type int? to type int.
